march_gazebo_plugins/src/obstacle_controller.cpp

```diff
@@ -58,14 +58,14 @@ ignition::math::v4::Vector3<double> ObstacleController::GetCom()
   return com / this->mass;
 }
 
-void ObstacleController::update(ignition::math::v4::Vector3<double>& torque_all,
-                                ignition::math::v4::Vector3<double>& torque_stable)
+void ObstacleController::update(ignition::math::v4::Vector3<double>& torque_left,
+                                ignition::math::v4::Vector3<double>& torque_right)
```

Comment:
Why pass these variables to the function if they remain unused? Between line 100-109 the variables are populated so maybe just create them locally instead of passing these as arguments.

Reply:
I use the variables as return objects since you cannot return two values (or you would have to construct an object that contains both)